sources/Applications/Web/Angular/Apps.Web.Angular.csproj

```diff
@@ -15,6 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.4" />
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.4" />
   </ItemGroup>
 
```

Comment:
Add another ItemGroup like 
<Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish" Condition="'$(Configuration)'=='Debug'">

Reply:
Something like this. Right?
 
` <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="3.1.4" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.4" />
  </ItemGroup>`